Academia-750/academia750-frontend-vue
# DBT-721 Upload Files to Cloudinary storage provider

## Context


This PR includes changes that will help us store files on the Cloudinary as a storage provider



[Configure storage provider](https://www.notion.so/tianlu/Backend-Configure-storage-provider-66e26901f982490eb02c0607ec13d163?pvs=4)

## Test



- Login as an admin 
- Go to manage workspaces
- check a click on the materials of the workspace
- add material and add a file 
- if the file is uploaded the cloud Icon on the material turns blue
- make sure you test that the file is found in Cloudinary



## Platform


Google chrome

## Proof of Concept
<img width="1680" alt="Screenshot 2023-08-10 at 09 35 35" src="https://github.com/Academia-750/academia750-frontend-vue/assets/79788219/3589a5bd-08c1-40d3-afc3-4d30c20a477b">
